wcvendors/wcvendors
# Added: Vendor ID and Product Count Columns

### All Submissions:


### Changes proposed in this Pull Request:

Adds 2 new columns and removes the posts column from the vendor role screen. 

Closes #857

### How to test the changes in this Pull Request:

1. Check User's screen go to vendor role
2. See new columns 
3. See posts column removed.
